lib/segment/src/types.rs

```diff
@@ -1196,6 +1196,32 @@ impl GeoRadius {
     }
 }
 
+/// Geo filter request
+///
+/// Matches coordinates inside the polygon, defined by the given coordinates in order.
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone, PartialEq)]
```

Comment:
I propose to remove the `JsonSchema` derivation and introduce it with the API to remind us to update the schema.